package.json

```diff
@@ -211,6 +211,7 @@
     "resolve-from": "5.0.0",
     "rimraf": "3.0.2",
     "sass": "1.54.0",
+    "satori": "0.4.4",
```

Comment:
This might mismatch with the Satori version inside `@vercel/og`, is it only for typing?

Reply:
yes, only for typing